views/js/submitScore.js

```diff
@@ -0,0 +1,18 @@
+var scorePopUp = function () {
+  var name = prompt("Please enter your name:", "anonymous");
+  var score = getScore();
+  console.log(score);
+  const xhr = new XMLHttpRequest();
+  const data = `name=${name}&score=${score}`
+  xhr.open("POST", "/scores/pacman/")
+  xhr.setRequestHeader("Content-Type", "application/x-www-form-urlencoded");
+  xhr.send(data);
+}
+
+var getScore = function () {
+  // TODO: retrieve score from pacman game instead of returning fake value
+  // var iframe = document.getElementById("kubercade_iframe");
+  // var iframeContent = iframe.contentWindow;
```

Comment:
You want `contentDocument` if I read this stackoverflow question correctly:

https://stackoverflow.com/questions/926916/how-to-get-the-bodys-content-of-an-iframe-in-javascript

Reply:
After doing a few other things this worked to get the score out of the iframe - fb33e27